### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by Jessekay**
**ON THIS MILESTONE I IMPLEMENTED THE FOLLOWING:**

- Page titles: No issues found
- Image text alternatives: Add the alternatives to images
- Text headings: Changed text headings, some were changed to h1 others to h2 and h3
- Color contrast: No issues found
- Resize: No issues found
- Interaction: No issues found
- Moving content: No issues found
- Multimedia: No issues found
- The basic structure of the page: No issues found

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,7 +12,7 @@
 <body>
   <header class="menu-wrap">
     <nav class="nav-bar">
-      <a href="#" class="brand">My Logo</a>
+      <a href="#" class="brand">Jesse</a>
```
Comment (souad988) [inline index.html:15]: good job so far @Jessekay , just make this logo color a bit darker as it's contrast ratio is not enough you can use the tools mentioned in the lesson to check if the contrast is good

File: index.html [hunk 2]
```diff
@@ -12,7 +12,7 @@
...
       <div class="hamburger">
```
Comment (souad988) [inline index.html:16]: Kindly add section tag to desktop navigation links make them links with `href=#section id`
 `<ul class="menu">
          <li>Portfolio</li>
          <li>About</li>
          <li>Contact</li>
        </ul>`

File: index.html [hunk 3]
```diff
@@ -35,17 +35,17 @@ <h1 class="title">I'm Jesse Glad to see you!</h1>
     <a href="#" class="connect">LET'S CONNECT</a><br>
```
Comment (souad988) [inline index.html:35]: Also please check contrast for this one as well

**Review by souad988 — CHANGES_REQUESTED** [review]
Hi @Jessekay , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


212385281-4797c58a-d358-423a-aa7c-832c22203865


## Highlights
✔No linter issues 
✔Good PR title and summary

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Kindly check your headline section background image on mobile it doesn't show

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by wayungi — APPROVED** [review]
Hi @Jessekay , 
image
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- [ ] All linter checks passing ✔️ 
- [ ] Good commit messages ✔
- [ ] No merge conflicts ✔️ 
- [ ]  Correct GitHub flow ✔️ 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**